### PR TITLE
Fixed padding for AoC leaderboard

### PR DESCRIPTION
**Pull request by saiTama-max**
Now the lines are equally spaced, also added a horizontal line separating the headers

### CODE REVIEW

**Review by fisher60 — CHANGES_REQUESTED** [review]
I cannot approve this, it contains changes from another branch, consider rebasing/fixing this branch so we can continue with the PR.

File: bot/cogs/utilities.py [hunk 1]
```diff
@@ -52,6 +73,35 @@ def __init__(self, bot: Friendo) -> None:
...
+            final.append(f'{hours} hours')
+
+        if days:
+            final.append(f'{days} days')
+
+        if months:
+            final.append(f'{months} months')
+
+        if years:
+            final.append(f'{years} years')
```
Comment (Xithrius) [inline bot/cogs/utilities.py:100]: I think most of this could be turned into a for loop that cheks items of a list.

File: bot/cogs/utilities.py [hunk 2]
```diff
@@ -2,22 +2,43 @@
 from pathlib import Path
 import random
 import subprocess
+import datetime
```
Comment (Xithrius) [inline bot/cogs/utilities.py:5]:
```suggestion
from datetime import datetime
```
You only use `datetime.datetime` for `utcnow()`, I don't think adding an extra layer is necessary.

File: bot/cogs/utilities.py [hunk 3]
```diff
@@ -183,6 +233,77 @@ async def quotes(self, ctx: Context) -> None:
...
+                           inline=False)
+        info_emb.add_field(name="Guild Profile",
+                           value=f"Joined: {joined_time} ago\nNick: {nick}\nRoles: {roles}",
+                           inline=False)
+        info_emb.add_field(name="Status",
+                           value=statuses,
+                           inline=False)
+        info_emb.add_field(name="Activity",
+                           value=activity,
+                           inline=False)
```
Comment (Xithrius) [inline bot/cogs/utilities.py:303]: You can combine all add_field calls into a for loop. The information would be stored in a dictionary for ease of editibility in the future.
```suggestion
        user_info = {
                'User Information': f'Created: {create_time} ago\nProfile: {men}\nID: {id_}\n{is_bot}',
                'Guild Profile': f'Joined: {joined_time} ago\nNick: {nick}\nRoles: {roles}',
                'Status': statuses,
                'Activity': activity
       }
       for k, v in user_info.items():
                info_emb.add_field(name=k, value=v, inline=False)
```

File: bot/cogs/utilities.py [hunk 4]
```diff
@@ -2,22 +2,43 @@
...
 VALID_PERIODS = "s sec secs second seconds m min mins minute minutes h hour hours".split()
 
+# Dictionaries for the emojis in userinfo embed
+BADGES = {'hypesquad_bravery': '<:bravery:784430728683323392>',
+          'hypesquad_brilliance': '<:brilliance:784430705118806036>',
+          'hypesquad_balance': '<:balance:784430681857720332>',
+          'nitro': '<:nitro:784751384780079134>',
+          'verified_bot_developer': '<:dev:784755774781390849>',
+          'partner': '<:partner:784756644634034176>',
+          'early_supporter': '<:early:784756624509108224>'}
```
Comment (Xithrius) [inline bot/cogs/utilities.py:30]: Probably could go into some constants/config (yaml) file.

**Review by Xithrius — CHANGES_REQUESTED** [review]

**Comment by Xithrius** [comment]
Discussion will continue in PR #178

**Pull request closed without merging**